pkgs/applications/editors/emacs-modes/elpa-packages.nix

```diff
@@ -25,7 +25,7 @@ self:
     super = removeAttrs imported [ "dash" ];
 
     elpaBuild = import ../../../build-support/emacs/elpa.nix {
-      inherit fetchurl lib stdenv texinfo;
```

Comment:
This is fixed in master, by @adisbladis: https://github.com/NixOS/nixpkgs/pull/43998#issuecomment-406969066